### PR TITLE
status bar padding issue fix

### PR DESCRIPTION
**Pull request by itscutefox**
values from havoc for alioth (ref: https://github.com/Havoc-Devices/android_device_xiaomi_alioth/blob/eleven/overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml#L25 )

### CODE REVIEW

File: overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
+<!-- These resources are around just to allow their values to be customized
+     for different hardware and product builds. -->
 <resources>
-    <dimen name="rounded_corner_content_padding">52px</dimen>
+
+    <!-- the padding on the top of the statusbar (usually 0) -->
+    <dimen name="status_bar_padding_top">8dp</dimen>
+
+    <!-- the padding on the rouded corner content -->
+    <dimen name="rounded_corner_content_padding">6dp</dimen>
```
Comment (peterbetos) [inline overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml]: Return the value

File: overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
+
+    <!-- the padding on the start of the statusbar -->
+    <dimen name="status_bar_padding_start">15dp</dimen>
```
Comment (peterbetos) [inline overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml]: Remove the line you are not sure is used on the system image

File: overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml [hunk 3]
```diff
@@ -1,20 +1,44 @@
...
+
+    <!-- the padding on the end of the statusbar -->
+    <dimen name="status_bar_padding_end">18dp</dimen>
```
Comment (peterbetos) [inline overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml]: Remove the line you are not sure is used on the system image

File: overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml [hunk 4]
```diff
@@ -1,20 +1,44 @@
...
+
+    <!-- the padding on the end of the keyguard statusbar -->
+    <dimen name="system_icons_keyguard_padding_end">15dp</dimen>
```
Comment (peterbetos) [inline overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml]: Remove the line you are not sure is used on the system image

File: overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml [hunk 5]
```diff
@@ -1,20 +1,44 @@
...
+    
+<!--     <dimen name="qqs_padding_top">20dp</dimen> -->
+
+    <!--  Allow CornerHandleView and PathSpecCornerPathRenderer to decouple from corner-radius -->
+    <dimen name="config_rounded_mask_size">113px</dimen>
```
Comment (peterbetos) [inline overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml]: Remove the line you are not sure is used on the system image

File: overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml [hunk 6]
```diff
@@ -1,20 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
- * Copyright (c) 2006, The Android Open Source Project
```
Comment (peterbetos) [inline overlay/frameworks/base/packages/SystemUI/res/values/dimens.xml]: Dont't change the license detail if you are not sure

**Review by peterbetos — CHANGES_REQUESTED** [review]

**Comment by itscutefox** [comment]
@peterbetos edited.

**Review by peterbetos — APPROVED** [review]
Saw the original property on AOSP. Approving this.

**Pull request closed without merging**